cronjob.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: gpu-stress-test
+spec:
+  schedule: "*/10 * * * *"
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          affinity:
+            nodeAffinity:
+              requiredDuringSchedulingIgnoredDuringExecution:
+                nodeSelectorTerms:
+                - matchExpressions:
+                  - key: resource.gpu
```

Comment:
Does this conform with the way the plugin is run with `runplugin` ?

https://github.com/waggle-sensor/surya-tools/blob/master/_helper_scripts/nx/_sampler_goals.json#L24

Reply:
I remember Sean added that label to NXs, so it should work with that label.

Reply:
Oh, I thought you asked about the label. No, `runplugin` does not run plugins as a cronjob. I know it is tricky, but at this time we need the plugin run indefinitely (with a sleep). Once we have round-robin scheduler the plugin may run this and finish.

Reply:
Oh okay. Yeah I will have to figure out how I want to run this in the surya lap. ill probably just git clone and deploy the yaml.

Reply:
Let me know if you need me to make any other changes to make it work in your workflow.

Reply:
I _think_ this is ok - I've never used that particular requiredDuring... field, though. I assume it does something similar to what's used by runplugin.

The dev notebook yaml uses exactly what runplugin does. You can compare to this: https://github.com/waggle-sensor/waggle-edge-stack/blob/c94d23d45b6731ddc5e59e67111f7a2bbfc7d8be/kubernetes/wes-dev-notebook.yaml#L70